kazzmir/paintown
# Build PSP

## Features
- [x] Create a docker file for cross compiling
- [x] Create script easy-compile-psp for building paintown-psp-$version.pbp
- [x] Create release-psp for building pbp locally
- [x] GitHub action with CI/CD for building releases
- [ ] Compile sdl2 and dependencies inside ps3toochain container
- [ ] Create meson cross file specific for PSP

## How to test

1. Download the latest paintown [pbp](https://github.com/kazzmir/paintown/releases/) from releases
2. Download and open [PPSSPP](https://www.ppsspp.org/download/) emulator specific for your current processor [x86,arm]
3. Install on PPSSPP the pbp and play the game

<img width="1401" alt="image" src="https://github.com/kazzmir/paintown/assets/9255997/62899488-27b6-48c4-ad92-3779efd8c896">

> [!NOTE]  
> Tested on Mac OS - ARM. But should work on any platform supported by PPSSPP
